src/TokenizerInput.react.js

```diff
@@ -99,7 +99,7 @@ const TokenizerInput = React.createClass({
           onChange={this._handleChange}
           onFocus={this.props.onFocus}
           onKeyDown={this._handleKeydown}
-          placeholder={selected.length ? null : placeholder}
```

Comment:
I think this is the only thing that needs to change for the `placeholder` change.